src/main/java/seedu/address/model/event/Time.java

```diff
@@ -42,7 +42,7 @@ public Time(String timeInput) {
     /**
      * Constructs a {@code Time}.
      *
-     * @param timeInput A valid time.
+     * @param timeInput A valid {@link LocalDateTime}.
```

Comment:
not `{@code}`?

Reply:
{@link} refers to a class and it means u can hover over it and see what it is in Intellij. {@code} can refer to any parameter or field.

Reply:
ah okay. thanks for clarifying.